src/contracts/core/RewardsCoordinator.sol

```diff
@@ -280,12 +309,28 @@ contract RewardsCoordinator is
         uint16 split
     ) external onlyWhenNotPaused(PAUSED_OPERATOR_PI_SPLIT) checkCanCall(operator) {
         uint32 activatedAt = uint32(block.timestamp) + activationDelay;
-        uint16 oldSplit = _getOperatorSplit(operatorPISplitBips[operator]);
-        _setOperatorSplit(operatorPISplitBips[operator], split, activatedAt);
+        uint16 oldSplit = _getOperatorSplit(_operatorPISplitBips[operator]);
+        _setOperatorSplit(_operatorPISplitBips[operator], split, activatedAt);
 
         emit OperatorPISplitBipsSet(msg.sender, operator, activatedAt, oldSplit, split);
     }
 
+    /// @inheritdoc IRewardsCoordinator
+    function setOperatorDirectedOperatorSetSplit(
```

Comment:
to clarify, in the sidecar is the level of prescedence:

1. operatorSet split
if not set, then
2. avs split
if not set, then
3. default